odoo/odoo
# [REF] mail, *: clean click and contains API

\* = test_mail, hr, im_livechat

- Allow only selectors
- Add `value` check to `contains`
- Remove return from `click`
